app/api/resources/grpc_client.py

```diff
@@ -85,4 +85,6 @@ def get_region(self, genome_uuid: str, region_name: str):
     def get_genome_uuid_from_slug(self, slug):
         uuid_request = ensembl_metadata_pb2.GenomeTagRequest(genome_tag=slug)
         genome_uuid_data = self.stub.GetGenomeUUIDByTag(uuid_request)
-        return genome_uuid_data.genome_uuid
\ No newline at end of file
+        if genome_uuid_data.genome_uuid:
+            return genome_uuid_data.genome_uuid
+        return slug
```

Comment:
You need to be able to handle garbage strings as well — you cannot assume that what the client sends you will always be a valid genome_tag or genome_uuid. Right now, this request:

https://staging-2020.ensembl.org/api/metadata/genome/just-kidding/details

will result in a beautiful 500 error:

<img width="1420" alt="image" src="https://github.com/Ensembl/ensembl-web-metadata-api/assets/6834224/98fa4622-e119-4cd2-82d3-289adcb08e24">

Reply:
Yes, I just finished working on this I just need to push this.

Reply:
By the way endpoint in question here is explain not genome_details :)

Reply:
```
curl 'http://localhost:8014/api/metadata/genome/just-kidding/explain' | jq
{
  "message": "Could not find details for just-kidding"
}
```

Reply:
@azangru  If you need specific message the do let us know and I will add it

Reply:
The message is fine. The code won't read it though. I expect http status code will be a 404? 🙂 

For the future — when we start having releases, I expect both request and the response will become more complex (e.g. genome id may exist, but not for a particular release).

Reply:
```
curl -sSD/dev/stderr 'http://localhost:8014/api/metadata/genome/just-kidding/explain' | jq
HTTP/1.1 404 Not Found
date: Tue, 31 Oct 2023 14:45:48 GMT
server: uvicorn
content-length: 53
content-type: application/json

{
  "message": "Could not find details for just-kidding"
}
```